### PR TITLE
Feature/config in package

### PR DESCRIPTION
**Pull request by hasruleng**
now we can use config file in all scenarios: 

1. source code run, 
2. package run, and 
3. testthat run

there are two config.yml that you can use to modify the constant variables:
1. in the root folder: for source code run & package run
2. in testthat folder: for testthat run

### CODE REVIEW

File: tests/testthat/test-quality_check.R [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 # Mock input data
-# dates <- config$TEST_FF_PREP_QC_DATE # Hasrul: we should find another way to use config file for our tests
-dates <- "2023-01-01"
-country <- "GAB"
+dates <- Sys.getenv("TEST_FF_PREP_QC_DATE")
+country <- Sys.getenv("TEST_FF_PREP_COUNTRY")
+datafolder <- Sys.getenv("TEST_DATA_FOLDER")
+#cat("From test-QC: the country is ")
+#cat(country)
```
Comment (aminbksi) [inline tests/testthat/test-quality_check.R:8]: did you run the tests to see they pass?

File: tests/testthat/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+URL_MARKDOWN: "https://raw.githubusercontent.com/jduijvenbode-WWF/ForestForesight/main/README.md"
```
Comment (aminbksi) [inline tests/testthat/config.yml:1]: why two config files? are they different?

File: man/config.Rd [hunk 3]
```diff
@@ -0,0 +1,187 @@
+% Generated by roxygen2: do not edit by hand
```
Comment (aminbksi) [inline man/config.Rd:1]: what is this file?

**Review by aminbksi — CHANGES_REQUESTED** [review]

**Review by aminbksi — APPROVED** [review]

File: tests/testthat/test-quality_check.R (hunk 1 above)
Comment (hasruleng) [reply tests/testthat/test-quality_check.R:8]: for this one, yes

**Pull request merged**